docker-alfresco/ags/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>content-services-docker-ags</artifactId>
+    <name>Alfresco Content Services Docker Image Builder for AGS</name>
+    <description>Alfresco Governance Services Enterprise Repository Extension</description>
+    <packaging>pom</packaging>
+
+    <parent>
+        <groupId>org.alfresco</groupId>
+        <artifactId>content-services-docker</artifactId>
+        <version>7.1.0-SNAPSHOT</version>
+    </parent>
+    <properties>
+        <image.name>alfresco/alfresco-governance-repository-enterprise</image.name>
+        <s3connector.version>3.1.0</s3connector.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-amps</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>org.alfresco</groupId>
+                                    <artifactId>alfresco-share-services</artifactId>
+                                    <version>${dependency.alfresco-community-repo.version}</version>
+                                    <type>amp</type>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.alfresco</groupId>
+                                    <artifactId>alfresco-governance-services-enterprise-rest-api-explorer</artifactId>
+                                    <version>${dependency.alfresco-enterprise-repo.version}</version>
+                                    <type>war</type>
+                                    <destFileName>gs-api-explorer-${project.version}.war</destFileName>
```

Comment:
The *gs-api-explorer* version can remain `${dependency.alfresco-enterprise-repo.version}`.